synapse/appservice/__init__.py

```diff
@@ -239,6 +209,21 @@ async def is_interested(self, event, store=None) -> bool:
 
         return False
 
+    @cached(num_args=1, cache_context=True)
+    async def is_interested_in_presence(
+        self, user_id: UserID, store, cache_context: _CacheContext
+    ):
```

Comment:
Please add a docstring! (And generally the same comments as for `matches_user_in_member_list`.)